llpc/test/shaderdb/partial_pipeline_cache_test/pipelines/Fs1.frag

```diff
@@ -0,0 +1,22 @@
+#version 450
+#extension GL_ARB_separate_shader_objects : enable
+
+layout(location = 0) out vec4 outColor;
+
+vec4 colors[3] = vec4[](
+  vec4(1.0, 0.0, 0.0, 1.0),
+  vec4(0.0, 1.0, 0.0, 1.0),
+  vec4(0.0, 0.0, 1.0, 1.0)
+);
+
+void main() {
+  outColor = colors[gl_SampleID%3];
+}
+
+// BEGIN_SHADERTEST
+// Fake test.
+// The real check will be included in the used pipelines.
+/*
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s
```

Comment:
1. Can you move this to the top of the file?
2. I see that the test is not strictly necessary (we could exclude this folder with a local lit config), but I wouldn't call it fake. I'd just say that this is not the main test and we only want to make sure that the shader is valid.
3. Check you check for `AMDLLPC SUCCESS`?

Reply:
Same for other shader files.

Reply:
Sure. I will fix this.